integration/src/test/resources/ammonite/integration/qs.scala

```diff
@@ -0,0 +1,13 @@
+def fib(n:Int):Int = {
+		if (n == 0) return 0
+		if (n == 1) return 1
+		return fib(n-1) + fib(n-2)
+	}
+
+def qs(unsorted:List[Int]):List[Int]={
+		if(unsorted.length<=1) unsorted
+		else{
+		val pivot=unsorted.head
+		return qs(unsorted.filter(_ < pivot)):::List(pivot):::qs(unsorted.filter(_ > pivot))
+		}
+	}
```

Comment:
Can you format this the same as the rest of our code? 

- One space after the `:`
- two-space indent
- no tabs
- Properly indented if-statements and closing parens
- Spaces around `=` and `<=` and `:::` and other operators
- `qs` should be called `quickSort`
- no explicit `return`s